src/apps/INTSC_HEXHEX-Cuda.cpp

```diff
@@ -0,0 +1,195 @@
+
+  Real_ptr vv_int_p = (Real_ptr ) vv_int + 8*blk ;
+
+  INTSC_HEXHEX_BODY;
+}
+
+
+template < Size_type block_size >
+__global__ void intsc_hexhex_fixup_vv_64to72
+    ( Real_ptr const vv_int,   // [8*intsc blks] blocked volumes, moments
+      Size_type const n_szpairs,  // number of subzone pairs
+      Real_ptr vv_pair )       // [4*n_szpairs] output voluments, moments
+{
+  Int_type i = blockIdx.x*block_size + threadIdx.x;
+  FIXUP_VV_BODY ;
+}
+
+
+
+template < Size_type block_size >
+void INTSC_HEXHEX::runCudaVariantImpl(VariantID vid)
+{
+  const Index_type run_reps = getRunReps();
+  const Index_type ibegin   = 0 ;
+  const Index_type iend     = tri_per_std_intsc * getActualProblemSize() ;
+
+  const Size_type  n_subz_intsc= npairs_per_std_intsc * getActualProblemSize();
+  const Size_type  nisc_stage  = n_subz_intsc ;
+
+  // n_szgrp is number of groups of subzone pairs in fixup kernel.
+  // gsize_fixup = fixup kernel grid size (1 thread per group of subzone pairs)
+  // iend_fixup = number of threads for fixup kernel.
+  //      Kernel has bounds check to mask out excess threads.
+
+  const Size_type  n_szgrp     =
+      RAJA_DIVIDE_CEILING_INT(n_subz_intsc, fixup_groupsize)  ;
+  const Size_type  gsize_fixup = RAJA_DIVIDE_CEILING_INT(n_szgrp, block_size) ;
+  const Index_type iend_fixup  = gsize_fixup * block_size ;
+
+  const Size_type  n_szpairs   = n_subz_intsc ;
+
+  auto res{getCudaResource()};
+
+  INTSC_HEXHEX_DATA_SETUP;
+
+  if ( vid == Base_CUDA ) {
+
+    startTimer();
+    for (RepIndex_type irep = 0; irep < run_reps; ++irep) {
+
+      const Size_type grid_size = RAJA_DIVIDE_CEILING_INT(iend, block_size);
+      constexpr Size_type shmem = 0;
+
+      RPlaunchCudaKernel( (intsc_hexhex<block_size>),
+                          grid_size, block_size,
+                          shmem, res.get_stream(),
+                          m_dsubz, m_tsubz,
+                          n_subz_intsc, m_vv_int ) ;
+
+      RPlaunchCudaKernel( (intsc_hexhex_fixup_vv_64to72<block_size>),
+                          gsize_fixup, block_size,
+                          shmem, res.get_stream(),
+                          m_vv_int, n_subz_intsc, m_vv_out ) ;
+
+
+    }
+    stopTimer();
+
+  } else if ( vid == Lambda_CUDA ) {
+
+    startTimer();
+    for (RepIndex_type irep = 0; irep < run_reps; ++irep) {
+
+      const Size_type grid_size = RAJA_DIVIDE_CEILING_INT(iend, block_size);
+
+      auto intsc_hexhex_lambda = [=] __device__
+          ( Index_type i )
+         {
+           __shared__ Real_type vv_reduce[len_vv_reduce] ;
+
+           Index_type blksize   = block_size ;
+           Index_type blk       = i / block_size ;
+           Index_type ith       = i ;
+           Index_type thridx    = i % block_size ;
+
+           Real_ptr vv_int_p = (Real_ptr ) vv_int + 8*blk ;
+           INTSC_HEXHEX_BODY; };
+
+      auto intsc_hexhex_fixup_lambda = [=] __device__
+          ( Index_type i )
+         {
+           FIXUP_VV_BODY ; } ;
```

Comment:
```suggestion
      auto intsc_hexhex_fixup_lambda = [=] __device__ ( Index_type i ) {
          FIXUP_VV_BODY ; 
      } ;
```

I think code formatting and alignment like this is easier to read. It's also consistent with implementations in other files throughout RAJA Perf.

Reply:
https://github.com/LLNL/RAJAPerf/pull/554/commits/1e9b2794ad78a35997fc9037a1bffe81e7fe2d8a